api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -798,6 +806,15 @@ type FelixConfigurationSpec struct {
 	// [Default: 1]
 	BPFExportBufferSizeMB *int `json:"bpfExportBufferSizeMB,omitempty" validate:"omitempty,cidrs"`
 
+	// FlowLogsPolicyEvaluationMode defines how policies are evaluated and reflected in flow logs.
+	// OnNewConnection - In this mode, staged policies are only evaluated when new connections are
```

Comment:
Remove reference to `OnNewConnection` and regenerate YAMLs

Reply:
Fixed.